src/controllers/RequestController.js

```diff
@@ -21,7 +21,7 @@ class RequestController {
     try {
       const { id } = req.decoded;
       const { body } = req;
-      const { dataValues } = await Request.create({ ...body, userId: id });
+      const { dataValues } = await Request.create({ ...body, UserId: id });
```

Comment:
why did you change userId to UserId

Reply:
Am waiting for your response to @chrismeeky 's question.

Reply:
I changed it because Sequelize was throwing an error that 'UserId' does not exist when I try to run the app. So when I changed 'userId' to UserId', the error stopped.